platform/api/gcloud/image.go

```diff
@@ -76,9 +76,16 @@ func (a *API) CreateImage(spec *ImageSpec, overwrite bool) (*compute.Operation,
 		Description: spec.Description,
 		Licenses:    licenses,
 		GuestOsFeatures: []*compute.GuestOsFeature{
+			// https://cloud.google.com/compute/docs/images/create-delete-deprecate-private-images
 			&compute.GuestOsFeature{
 				Type: "VIRTIO_SCSI_MULTIQUEUE",
 			},
+			&compute.GuestOsFeature{
+				Type: "UEFI_COMPATIBLE",
+			},
+			&compute.GuestOsFeature{
```

Comment:
@bgilbert do you know if CL+GCE supports this?

Reply:
UEFI should work.  Secure Boot does not.